core_backend/app/users/models.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+from typing import Optional
+
+from sqlalchemy import (
+    DateTime,
+    String,
+    # delete,
+    select,
+)
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Mapped, mapped_column
+
+from ..models import Base
+from .schemas import UserCreate
+
+
+class UserDB(Base):
+    """
+    SQL Alchemy data model for users
+    """
+
+    __tablename__ = "user"
+
+    user_id: Mapped[str] = mapped_column(String, primary_key=True, nullable=False)
+    username: Mapped[str] = mapped_column(String, nullable=False)
```

Comment:
Should there be a password here? Or sorted by google oauth?

Reply:
See previous comment. Hoping to defer passwords to OAuth.